### ISSUE TITLE
CSS filters on divs are rendering black

### ISSUE THREAD

**Issue by audas** [RPi-Distro/chromium-browser#10]
This is failing with the latest kernel 5.10 and native gl renderer 

dtoverlay=vc4-kms-v3d.

This was an issue in Chrome several years ago with Nividia - removing chrome://flags use GPU hardware rendering removed the problem.

**Comment by XECDesign** [comment]
@naushir One for Igalia?

**Comment by popcornmix** [comment]
@audas can you provide a url that exhibits the issue?

**Comment by audas** [comment]
https://display-screen.media/display.php?id_presentations_manifests=16

The images at the top and "education funding" third from bottom - these are being animated VERY SLOWLY to give a chance to inspect them. They are all the same text etc. Top images should be transparent background.

The reason there is a difference between other texts is the filter and these are all divs within divs. But again - when I switched off hardware acceleration in flags in chrome it cleared this up. (I need hardware acceleration though).

I just changed the chromium-browser back to 78 and it fixed the issue.
